filestore/path_only_handlers.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import six
+
+import logging
+
+from .retrieve import HandlerBase
+
+logger = logging.getLogger(__name__)
+
+
+class AreaDetectorTiffPathOnlyHandler(HandlerBase):
+
```

Comment:
It would be very helpful if you could add an Example docstring heading with the following code

```
>>> # Use this PathOnly handler
>>> import filestore
>>> from filestore import path_only_handlers
>>> # Show the currently registered handlers
>>> filestore.retrieve._h_registry
{u'AD_HDF5': filestore.handlers.AreaDetectorHDF5Handler,
 u'AD_SPE': filestore.handlers.AreaDetectorSPEHandler,
 u'AD_TIFF': filestore.path_only_handlers.AreaDetectorTiffPathOnlyHandler,
 u'MAPS_PLANE': filestore.handlers.HDFMapsEnergyHandler,
 u'MAPS_SPECTRUM': filestore.handlers.HDFMapsSpectrumHandler,
 u'npy': filestore.handlers.NpyHandler,
 u'npy_FRAMEWISE': filestore.handlers.NpyFrameWise}
>>> # Overwrite the AD_TIFF handler with this PathOnly handler
>>> filestore.retrieve.register_handler('AD_TIFF', path_only_handlers.AreaDetectorTiffPathOnlyHandler, overwrite=True)
```